doc/plugins/proxy-rewrite-cn.md

```diff
@@ -0,0 +1,72 @@
+[English](proxy-rewrite.md)
+# proxy-rewrite
+
+上游代理信息重写插件。
+
+#### 配置参数
+|名字    |可选|说明|
+|-------         |-----|------|
+|scheme          |可选| 转发到上游的新`schema` 协议，可以是`http`或`https`，默认`http`协议|
+|uri             |可选| 转发到上游的新`uri` 地址|
+|host            |可选| 转发到上游的新`host` 地址，例如：`iresty.com` |
+|enable_websocket|可选| 是否启用`websocket`（布尔值），默认不启用|
+
+### 示例
+
+#### 开启插件
+下面是一个示例，在指定的 route 上开启了 proxy rewrite 插件:
```

Comment:
`proxy rewrite` is a special noun, so it's best to wrap it around with \`, example: \`xxxx\`